test.ini

```diff
@@ -17,6 +17,8 @@ solr_url = http://127.0.0.1:8983/solr
 # tests here.
 ckan.plugins = xloader
 ckanext.xloader.jobs_db.uri = sqlite:////tmp/jobs.db
+ckanext.xloader.max_content_length = 70000
+ckanext.xloader.max_excerpt_lines = 100
```

Comment:
I hope there is a better way to mock these configuration-options in my tests, haven't gotten around to find that out. 
This is meant to demonstrate, though, how the tests and most of all the functionality works now.